charts/shibboleth-idp/templates/service.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: 80
+      targetPort:  {{ .Values.service.containerPort }}}}
```

Comment:
I think this needs to be updates in the deployment.yaml as well: https://github.com/MiamiOH/helm-shibboleth-idp/blob/main/charts/shibboleth-idp/templates/deployment.yaml#L38